src/main/java/com/dongtronic/diabot/commands/nightscout/NightscoutCommand.kt

```diff
@@ -67,7 +68,7 @@ class NightscoutCommand(category: Command.Category) : DiabotCommand(category, nu
 
     }
 
-    private fun buildNightscoutResponse(endpoint: String, token: String?, avatarUrl: String?, event: CommandEvent) {
+    private fun buildNightscoutResponse(endpoint: String, token: String?, displayOpts: String, avatarUrl: String?, event: CommandEvent) {
```

Comment:
Please call the display options string `displayOptions` everywhere. Currently I see it being called both that and `displayOpts` which is confusing.